packages/itwin/measure-tools/src/api/SheetMeasurementHelper.ts

```diff
@@ -159,16 +159,50 @@ export namespace SheetMeasurementsHelper {
    * @param allowedDrawingTypes
    * @returns
    */
-  export function checkIfAllowedDrawingType(ev: BeButtonEvent, allowedDrawingTypes: DrawingType[]) {
-    if (!ev.viewport)
+  export function checkIfAllowedDrawingType(viewport: ScreenViewport | undefined, point: Point3d, allowedDrawingTypes: DrawingType[]) {
+    if (!viewport)
       return false;
-    for (const drawing of DrawingDataCache.getInstance().getSheetDrawingDataForViewport(ev.viewport)) {
+    for (const drawing of DrawingDataCache.getInstance().getSheetDrawingDataForViewport(viewport)) {
       if (allowedDrawingTypes.includes(drawing.type)) {
-        if (SheetMeasurementsHelper.checkIfInDrawing(ev.point, drawing.origin, drawing.extents)) {
+        if (SheetMeasurementsHelper.checkIfInDrawing(point, drawing.origin, drawing.extents)) {
           return true;
         }
       }
     }
     return false;
   }
+
+  function getNameFromDrawingType(type: SheetMeasurementsHelper.DrawingType): string  {
+    if (type === SheetMeasurementsHelper.DrawingType.CrossSection) {
+      return IModelApp.localization.getLocalizedString("CivilReviewTools:SheetMeasurementTooltip.MeasurementNames.CrossSection");
+    } else if (type === SheetMeasurementsHelper.DrawingType.Plan) {
+      return IModelApp.localization.getLocalizedString("CivilReviewTools:SheetMeasurementTooltip.MeasurementNames.Plan");
+    } else if (type === SheetMeasurementsHelper.DrawingType.Profile) {
+      return IModelApp.localization.getLocalizedString("CivilReviewTools:SheetMeasurementTooltip.MeasurementNames.Profile");
+    } else {
+      return "";
+    }
+  }
+
+  export async function getSheetErrorToolTipText(hit: HitDetail, allowedDrawingTypesList: SheetMeasurementsHelper.DrawingType[], defaultToolTip:(hit: HitDetail) => Promise<HTMLElement | string>): Promise<string | HTMLElement> {
```

Comment:
The suggestion for changing the name was an "or" with the suggestion of bumping the usage logic to the base measurement tool. Since you did the latter (and it all looks good), I wouldn't have changed the helper name, since if the types are allowed, it's not returning an error. It's a nitpick. I'd just keep the name "getSheetToolTipText" or "getSheetToolTipTextOrError".